provisioning/standalone/docs/example.md

```diff
@@ -0,0 +1,94 @@
+# Example
+
+In this example, we are going to learn how to deploy a simple echo server using **golang**, which will answer to a **GET HTTP request** with a predefined message.
+
+### Create the server
+
+Create a **main.go** file which will contain the source code of the server and **copy/paste** the following code inside **main.go**
+
+```go
+import (
+	"fmt"
+	"net/http"
+	"os"
+)
+
+func hello(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Hello!\n")
+}
+
+func main() {
+	port := os.Args[1]
+	basepath := os.Args[2]
+	http.HandleFunc(fmt.Sprintf("%s/", basepath), hello)
+	err := http.ListenAndServe(fmt.Sprintf("0.0.0.0:%s", port), nil)
+	if err != nil {
+		fmt.Println(err)
+	}
+}
+```
+
+Let's test it by running it with `go run 8888 /hello`.
+
+In the meanwhile open another terminal and launch the command `curl localhost:8888/hello`, and you will get `Hello!` as response.
+
+### Build the container
+
+Let's create a **Dockerfile** ` and **copy/paste** the following code inside of it.
```

Comment:
```suggestion
Let's create a **Dockerfile**, and **copy/paste** the following code inside of it.
```

Should add a brief description about what the Dockerfile does, otherwise a reader would not know what to pay attention to.

Reply:
I added some comments in the Dockerfile, tell me if it's clear and enough.

Reply:
I would personally write something like:

```
Let's create a **Dockerfile** to build and package the HTTP server, and **copy/paste** the following code inside of it.
```